Jdp190401 25

pszczepcio opened the pull request.

pszczepcio committed:
- dodanie nowych zaeźności

pszczepcio committed:
- nadpisanie metody findAll w dao
- stworzenie userService
- stworzenie kontrolera
- dodanie klas Dto
- stworzenie userMapper

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+        return changeStatus;
+
+    }
+    private String newPassword(final UpDateAccount upDateAccount, boolean activeKey) throws ParseException {
```
Moze umiesc SystemOut zwracajacy jakies info na temat poprawnej aktualizacji password w metodzie updatePassword

Makedo81 replied:
private String createNewPassword()

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+        return userDto;
+    }
+
+    private boolean newStatus(final ChangeStatusByAdmin changeStatusByAdmin) throws ParseException {
```
ChangeStatusByAdmnin - > modifiedAdminStatus, chociaz nie do konca mi pasuje ta nazwa , moze ktos inny cos zaproponuje :)

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+    }
+
+    private boolean newStatus(final ChangeStatusByAdmin changeStatusByAdmin) throws ParseException {
+        boolean changeStatus = false;
```
boolean statusIsChanged

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+        return info;
+    }
+
+    private boolean validityKey(UpDateAccount upDateAccount){
```
private boolean validateKey()

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+
+    private boolean validityKey(UpDateAccount upDateAccount){
+        UserDto userDto = new UserDto();
+        Calendar finshKeyValidity = Calendar.getInstance();
```
finshKeyValidity - > keyExpiredTime

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+        return activeKey;
+    }
+
+    private boolean checkAccount(AccauntDto accauntDto){
```
AccountDto ->UserAccountDto

Makedo81 replied:
private boolean checkAccount() -> boolean verifyUserAccount()

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+        boolean checkAccount = false;
+        if (userDao.findById(accauntDto.getUserId()).get().getPassword().equals(accauntDto.getPassword()) &&
+                userDao.findById(accauntDto.getUserId()).get().getUserKey() == accauntDto.getKey()){
+            checkAccount = true;
```
checkAccount -> accountIsVerified

Makedo81 commented on src/main/java/com/kodilla/ecommercee/domain/UpDateAccount.java:
```diff
@@ -0,0 +1,17 @@
+@NoArgsConstructor
+@Setter
+@Getter
+public class UpDateAccount {
```
UpDateAccount -> UpdatedAccount

Makedo81 reviewed (changes requested):
Kilka poprawek odnosnie czytelnosci kodu

Makedo81 commented on src/main/java/com/kodilla/ecommercee/service/UserService.java:
```diff
@@ -1,4 +1,154 @@
+        if(userDao.findById(upDateAccount.getUserId()).get().getPassword().equals(upDateAccount.getPassword()) &&
+                userDao.findById(upDateAccount.getUserId()).get().isStatus() == true &&
+                Calendar.getInstance().before(finshKeyValidity)){
+                activeKey = true;
```
activeKey -> keyIsActive

pszczepcio committed:
- zmiana nazewnictwa w userService
- dołożenie w userDto dwóch List : List<CartDto> listDtoCart, List<OrderDto> orderDtoList
- dodanie metody createFirstKey - generującej pierwszy key przy zakłądaniu konta

pszczepcio commented:
Wszystkie uwagi zostały poprawione.

Makedo81 reviewed (approved):
Nie widze jakis konkretnych bledow w kodzie, warunki w metodach sa w porzadku
